src/ControlSystem/ControlErrors/Size.hpp

```diff
@@ -404,7 +408,8 @@ struct Size : tt::ConformsTo<protocols::ControlError> {
             ? std::optional<double>(control_error_delta_r -
                                     delta_r_drift_outward_options_.value()
                                         .outward_drift_velocity -
-                                    (lambda_00 + horizon_00 -
+                                    (lambda_00 +
+                                     spherepack_factor * horizon_00 -
```

Comment:
Can you add a test that would have caught this?

Reply:
Testing this is extremely difficult because it's just in the mess that's the size control error. The test currently turns off the `delta_r_drift_outward` because there's not a good way to know the expected value of the size control error if we turn it on. Also, it's very hard to come up with parameters such that we transition to the correct state that this quantity will be used in.

So in short, no not really, unless I spend days on this single test.

Reply:
Ok @nilsvu I added a test